hack/common.sh

```diff
@@ -128,9 +132,18 @@ os::build::build_binaries() {
     for platform in "${platforms[@]}"; do
       os::build::set_platform_envs "${platform}"
       echo "++ Building go targets for ${platform}:" "${targets[@]}"
-      go install "${goflags[@]:+${goflags[@]}}" \
-          -ldflags "${version_ldflags}" \
-          "${binaries[@]}"
+      for b in ${binaries[@]}; do
+          echo "Building $b"
+          if [[ "$b" == *"e2e.test"* ]]; then
```

Comment:
The goal is to publish an e2e alongside all other artifacts.  Of course we can debate wether thats the right goal or not :).  but right now thats the goal of the patch.